### DISCUSSION TITLE
Custom Music

### DISCUSSION THREAD

**Discussion by Cyancolt** [26F-Studio/Techmino#1114] [Ideas]
Im completely new to this, but as a tetr.io planning on moving over to techmino, i see there are custom backgrounds, but no custom music. is there a way to add it that I missed? If not, can it be added as a feature?

**Comment by MrZ626** [comment]
you can just play your music in background, there's totally no difference, so custom music won't be added as feature

**Comment by shoucandanghehe** [comment]
As #1068 said
